erpnext/projects/doctype/project/project.py

```diff
@@ -772,3 +778,29 @@ def recalculate_project_total_purchase_cost(project: str | None = None):
 			"total_purchase_cost",
 			(total_purchase_cost and total_purchase_cost[0][0] or 0),
 		)
+
+
+def calculate_total_ordered_cost(project: str | None = None):
```

Comment:
method name should be get_total_ordered_cost which should return numeric value and not list

Reply:
Agreed, it's an inspirated code from exiting [calculate_total_purchase_cost](https://github.com/frappe/erpnext/blob/c5f22e76165aaf4132e94246f84dc888f55be5cb/erpnext/projects/doctype/project/project.py#L732) function.
To be consistent I change in this PR the exiting method name, is it OK ?